hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java

```diff
@@ -1770,6 +1772,7 @@ private long prepareRegionForBachPut(final Put[] puts, final MetricsWALSource so
   // checkAndMutate tests
   // ////////////////////////////////////////////////////////////////////////////
   @Test
+  @Deprecated
```

Comment:
Probably don't need to mark deprecated tests in the future, but this is fine to leave, IMO.